lib/arena.rb

```diff
@@ -1,2 +1,42 @@
+# require "pry"
+
+require_relative "gladiator"
 class Arena
+
+  attr_accessor :name, :gladiators
+
+  def initialize(name)
+    @name = name.capitalize
+    @gladiators = []
+  end
+
+  def add_gladiator(gladiator)
+    if @gladiators.length < 2
+      @gladiators << gladiator
+    end
+  end
+
+  def fight
+    return if @gladiators.length < 2
+
+      if @gladiators[0].weapon == "Spear" && @gladiators[1].weapon == "Trident"
+          @gladiators.shift
+        elsif @gladiators[1].weapon == "Spear" && @gladiators[0].weapon == "Trident"
+          @gladiators.pop
+        elsif @gladiators[0].weapon == "Club" && @gladiators[1].weapon == "Trident"
+          @gladiators.pop
+        elsif @gladiators[1].weapon == "Club" && @gladiators[0].weapon == "Trident"
```

Comment:
Good job on this method here for fight, could you think of a way to achieve the same with fewer lines of code?